css/styles.css

```diff
@@ -235,3 +235,30 @@ li {
 .hidden {
   display: none;
 }
+
+.visually-hidden {
+  /* Remove the item from normal flow */
```

Comment:
Next time when you use some AI tools don't forget to remove obsolete comments :)

Reply:
Haha, I copied it from the module [materials](https://d17btkcdsmqrmh.cloudfront.net/webaccessibility/docs/module-2/styles#9-hiding-content-visually) 😆